.circleci/config.yml

```diff
@@ -227,9 +240,16 @@ workflows:
           filters:
             branches:
               ignore: /^master?/
-      - run_pp:
-          name: "run pp tests"
+      - run_premium_primitives_tests_on_master:
+          name: "run premium primitives tests on master"
           filters:
             branches:
               only:
-                - master
\ No newline at end of file
+                - master
+      - run_premium_primitives_tests_on_release:
+          name: "run premium primitives tests on release"
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: ^v\d+\.\d+\.\d+$
```

Comment:
does this need to be surrounded by forward slashes?

Reply:
Yes you do. If you don't put the slash (`\`) it means that it will look for the letter `d` rather than the digit between 0-9.

Reply:
No I mean shouldn't it be `/^v\d+\.\d+\.\d+$/`

Reply:
Fixed